guides/resource-provisioning/servicebus-queues.md

```diff
@@ -0,0 +1,138 @@
+# Azure Service Bus Queues
+
+This guide describes how to add Azure Service Bus queues to a microservice running on the FFC Platform Kubernetes cluster.
+
+## Create a microservice Managed Identity
+
+If not alreay configured, [add an Azure Managed Identity to your microservice](managed-identity.md).
+
+## Request creation of Service Bus queues
+
+For each queue required by your microservices, you will need:
+* A Service Bus queue for the main branch deploy
+* A Service Bus queue for each of the developers on the workstream to use for local development
```

Comment:
Should specify these developer queues will be in sandpit subscription

Reply:
Fixed